src/Form/EthereumServerForm.php

```diff
@@ -84,14 +84,12 @@ public function form(array $form, FormStateInterface $form_state) {
   public function save(array $form, FormStateInterface $form_state) {
     $server = $this->entity;
     $status = $server->save();
-
     if ($status == SAVED_UPDATED) {
-      $this->messenger()->addStatus($this->t('The server %label has been updated.', ['%label' => $server->label()]));
+      drupal_set_message($this->t('The server %label has been updated.', ['%label' => $server->label()]));
```

Comment:
`drupal_set_message()` is deprecated, the previous code is using the proper API.